src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
+/* eslint-disable max-len */
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  moviesData: Movie[],
+};
+
+export const MoviesList: React.FC<Props> = ({ moviesData }) => (
+  <div className="movies">
+    {moviesData.map(movieOfMoves => <MovieCard movie={movieOfMoves} key={movieOfMoves.imdbId} />)}
```

Comment:
let's not write code in one line, format it properly
```suggestion
    {moviesData.map((movieOfMoves) => (
      <MovieCard movie={movieOfMoves} key={movieOfMoves.imdbId} />
    ))}
```